ports/marl/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "marl",
-  "version-string": "2020-06-30",
+  "version-string": "2020-10-2",
```

```suggestion
  "version-string": "2020-10-10",
```
The version should be a date that you accept the commit.